src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -43,48 +74,60 @@ const DiskCharts = ({ vm, isRunning, id, ...props }) => {
   })
 
   const { unit, value: disks } = convertValueMap('B', { actualSize, provisionedSize })
-  const used = round(disks.actualSize, 1)
   const available = round(disks.provisionedSize - disks.actualSize, 1)
   const total = round(disks.provisionedSize, 1)
+  const used = round(disks.actualSize, 1)
 
   return (
     <UtilizationCard className={style['chart-card']} id={id}>
-      <CardTitle>Disk <span style={{ fontSize: '55%', verticalAlign: 'super' }}>(storage allocations)</span></CardTitle>
+      <CardTitle>Disk</CardTitle>
       <CardBody>
-        { !hasDisks &&
-          <NoLiveData id={`${id}-no-live-data`} message='This VM has no attached disks.' />
+        { !diskDetails && !isRunning &&
+          <NoLiveData id={`${id}-no-live-data`} />
         }
-        { hasDisks &&
-        <React.Fragment>
-          <UtilizationCardDetails>
-            <UtilizationCardDetailsCount id={`${id}-available`}>{available}</UtilizationCardDetailsCount>
-            <UtilizationCardDetailsDesc>
-              <UtilizationCardDetailsLine1>Unallocated</UtilizationCardDetailsLine1>
-              <UtilizationCardDetailsLine2 id={`${id}-total`}>of {total} {unit} Provisioned</UtilizationCardDetailsLine2>
-            </UtilizationCardDetailsDesc>
-          </UtilizationCardDetails>
-
-          <DonutChart
-            id={`${id}-donut-chart`}
-            data={{
-              columns: [
-                [`${unit} allocated`, used],
-                [`${unit} unallocated`, available],
-              ],
-              order: null,
-            }}
-            title={{
-              primary: `${used}`,
-              secondary: `${unit} Allocated`,
-            }}
-            tooltip={{
-              show: true,
-              contents: patternfly.pfDonutTooltipContents,
-            }}
-          />
-
-          {/* Disks don't have historic data */}
-        </React.Fragment>
+        { isRunning && vm.get('disks').size === 0 &&
+          <NoLiveData id={`${id}-no-live-data`} message='It looks like no disk is attached to VM.' />
+        }
+        { isRunning && vm.get('disks').size > 0 &&
+          <React.Fragment>
+            <UtilizationCardDetails>
+              <UtilizationCardDetailsCount id={`${id}-available`}>{available}</UtilizationCardDetailsCount>
+              <UtilizationCardDetailsDesc>
+                <UtilizationCardDetailsLine1>Unallocated</UtilizationCardDetailsLine1>
+                <UtilizationCardDetailsLine2 id={`${id}-total`}>of {total} {unit} Provisioned</UtilizationCardDetailsLine2>
+              </UtilizationCardDetailsDesc>
+            </UtilizationCardDetails>
+            { !diskDetails &&
```

Comment:
Are we sure that we want to keep the donut chart around for VMs without a guest agent?